great_expectations/datasource/data_connector/util.py

```diff
@@ -1,571 +0,0 @@
-    """  # noqa: E501
-    if iterator_dict is None:
-        iterator_dict = {}
-
-    if "continuation_token" in iterator_dict:
-        query_options.update({"ContinuationToken": iterator_dict["continuation_token"]})
-
-    logger.debug(f"Fetching objects from S3 with query options: {query_options}")
-
-    s3_objects_info: dict = s3.list_objects_v2(**query_options)
-
-    if not any(key in s3_objects_info for key in ["Contents", "CommonPrefixes"]):
-        raise ValueError("S3 query may not have been configured correctly.")  # noqa: TRY003
-
-    if "Contents" in s3_objects_info:
-        keys: List[str] = [item["Key"] for item in s3_objects_info["Contents"] if item["Size"] > 0]
-        yield from keys
-
-    if recursive and "CommonPrefixes" in s3_objects_info:
-        common_prefixes: List[Dict[str, Any]] = s3_objects_info["CommonPrefixes"]
-        for prefix_info in common_prefixes:
-            query_options_tmp: dict = copy.deepcopy(query_options)
-            query_options_tmp.update({"Prefix": prefix_info["Prefix"]})
-            # Recursively fetch from updated prefix
-            yield from list_s3_keys(
-                s3=s3,
-                query_options=query_options_tmp,
-                iterator_dict={},
-                recursive=recursive,
-            )
-
-    if s3_objects_info["IsTruncated"]:
-        iterator_dict["continuation_token"] = s3_objects_info["NextContinuationToken"]
-        # Recursively fetch more
-        yield from list_s3_keys(
-            s3=s3,
-            query_options=query_options,
-            iterator_dict=iterator_dict,
-            recursive=recursive,
-        )
-
-    if "continuation_token" in iterator_dict:
-        # Make sure we clear the token once we've gotten fully through
-        del iterator_dict["continuation_token"]
-
-
-# TODO: <Alex>We need to move sorters and _validate_sorters_configuration() to DataConnector</Alex>
-# As a rule, this method should not be in "util", but in the specific high-level "DataConnector" class, where it is  # noqa: E501
-# called (and declared as private in that class).  Currently, this is "FilePathDataConnector".  However, since this  # noqa: E501
-# method is also used in tests, it can remain in the present "util" module (as an exception to the above stated rule).  # noqa: E501
-def build_sorters_from_config(config_list: List[Dict[str, Any]]) -> Optional[dict]:
-    sorter_dict: Dict[str, Sorter] = {}
-    if config_list is not None:
-        for sorter_config in config_list:
-            # if sorters were not configured
-            if sorter_config is None:
-                return None
-
-            if "name" not in sorter_config:
-                raise ValueError("Sorter config should have a name")  # noqa: TRY003
-
-            sorter_name: str = sorter_config["name"]
-            new_sorter: Sorter = _build_sorter_from_config(sorter_config=sorter_config)
-            sorter_dict[sorter_name] = new_sorter
-
-    return sorter_dict
```

Comment:
if these utils were in use, the were migrated to the fluent dataconnector module where they are used